hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperator.java

```diff
@@ -257,7 +257,7 @@ private void onAddedOrUpdated() {
         if (backup) {
             recordStore.putBackup(dataKey, newValue, NOT_WAN);
```

Comment:
i think we also need to set ttl for backup-entry-processing case. Please addd another test for this case.

Reply:
Hi @ahmetmircik, I'm not sure how I get access to a backup `EntryView` or equivalent in order to test (and fix) this.

Reply:
Hi, as in this [test](https://github.com/hazelcast/hazelcast/blob/8d26f50b58571769e45ab0802b0d708f4a531cb0/hazelcast/src/test/java/com/hazelcast/map/BackupExpirationTest.java#L106), eventually, total entry count(owned + backup) should be zero. You can copy that test and can change `put` with `entry-processor` equivalent.

Reply:
Thanks @ahmetmircik. I did add a test that digs into the RecordStore to verify the record has the correct TTL. The test fails if I revert my change to apply backup TTLs. Let me know if that test is good or if you'd like the `eventually` style test. Personally I feel the test I wrote is more accurately testing the TTL is set correctly on the Record and should run a little quicker. But am open to feedback on what you'd rather see.